prefect_dbt/cli/configs/bigquery.py

```diff
@@ -80,7 +80,7 @@ class BigQueryTargetConfigs(TargetConfigs):
     _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/5zE9lxfzBHjw3tnEup4wWL/8cb73be51575a659667f6471a24153f5/dbt-bit_tm.png?h=250"  # noqa
     _description = "dbt CLI target configs containing credentials and settings, specific to BigQuery."  # noqa
 
-    type: Literal["gcp"] = "gcp"
+    type: Literal["bigquery"] = "bigquery"
```

Comment:
Do you think we could make this attribute semi-private so that it doesn't display in the UI? We need the value, but I don't think it makes sense to display in the UI.

Reply:
After some conversation, I retract this comment since the `type` attribute is important to the `TargetConfigs` base class. Populating the default value in the UI will allow the user to not need to interact with the rendered dropdown.